### PR TITLE
Update proprietary code system page for multi-region support

### PR DESCRIPTION
**Pull request by xiaoningmzhang**
Description
----
Updated the Millennium R4 Proprietary codes and systems page for multi-region support.

Screen shots of the updates:
afterchange-1
afterchange-2
afterchange-3
afterchange-4

PR Checklist
----
- [x] Screenshot(s) of changes attached before changes merged.
- [ ] Screenshot(s) of changes attached after changes merged and published.

### CODE REVIEW

File: content/millennium/r4/proprietary-codes-and-systems.md [hunk 1]
```diff
@@ -13,11 +13,11 @@ title: Proprietary Codes and Systems | R4 API
...
 
-Millennium groups repetitive textual information into code sets. The code set stores a numeric code value that represents a textual or character display. Code sets are consistent across all clients. But the values in a code set vary between clients and are only guaranteed unique within a specific EHR system, which is the reason for the EHR source id (or tenant) qualifier in the system URL.
+Millennium groups repetitive textual information into code sets. The code set stores a numeric code value that represents a textual or character display. Code sets are consistent across all clients. But the values in a code set vary between clients and are only guaranteed unique within a specific EHR system, which is the reason for the region and the EHR source id (or tenant) qualifier in the system URL. See [Multi-Region Support](#multi-region-support) for further details.
```
Comment (cerbs) [inline content/millennium/r4/proprietary-codes-and-systems.md:16]: The original wording of this paragraph is kind of clunky. Maybe we could streamline it a little?

```suggestion
Millennium groups repetitive textual information into code sets. A code set stores numeric code values that represent textual or character displays. Code sets are consistent across all clients, however the values within a code set vary between clients and are only guaranteed unique within a specific EHR system. For this reason the region and the EHR source id (or tenant) are included in the system URL. See [Multi-Region Support](#multi-region-support) for further details.
```

**Comment by dennispatterson** [comment]
> There are examples of the system values for the proprietary codes on this page and a lot of other resource pages. I didn't change them to include the `<region> ` variable as it is optional. Should we change all of them to keep the consistency?

No, the default for US omits the region so the examples are still valid.

File: content/millennium/r4/proprietary-codes-and-systems.md [hunk 2]
```diff
@@ -873,8 +873,25 @@ This system is the synonym id for an order and the ingredients.
...
       'userSelected': false
     }
+
+## Multi-Region Support
+
+### Overview
+
+Proprietary codes and systems that are dependent on a specific EHR system may include a Cerner cloud region in their `system` value. The default `system` value format is `https://fhir.cerner.com/<EHR source id>/`.
+
+### Regions
```
Comment (dennispatterson) [inline content/millennium/r4/proprietary-codes-and-systems.md:888]:
```suggestion
### Region-specific Systems
```

File: content/millennium/r4/proprietary-codes-and-systems.md [hunk 3]
```diff
@@ -873,8 +873,25 @@ This system is the synonym id for an order and the ingredients.
...
+
+The `system` value formats for the following regions are:
```
Comment (dennispatterson) [inline content/millennium/r4/proprietary-codes-and-systems.md:890]:
```suggestion
The following regions override the format above with a region-specific `system`:
```

**Comment by xiaoningmzhang** [comment]
Resolved all the comments. See attached screenshots after the updates.
updated-proprietary-page-2
updated-proprietary-page-1

**Review by cerbs — APPROVED** [review]

**Review by dennispatterson — APPROVED** [review]

**Pull request merged**